sources/odoo.conf

```diff
@@ -0,0 +1,116 @@
+[options]
+
+
+; --------------------
+; change this ASAP!!!!
+;---------------------
+admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+
+; set the default timezone for openerp interface
+timezone = Europe/Paris
+
+db_maxconn = 64
+
+db_name = False
+db_template = template1
+dbfilter = .*
+
+; set this to True if you have installed unaccent in your PostgreSQL
+; and want to be able to search without accents
+unaccent = True
+
+; These options are really important and depend on the type of
+; deployment you want to have so read the doc carefully and
+; set these values accordingly
+workers = 0
+max_cron_threads = 2
+limit_memory_hard = 2147483648
```

Comment:
These options have a huge impact on Odoo depending on our server configuration. It would be nice if we could assign in our docker-compose.yml (ENV variable maybe).